### ISSUE TITLE
ngl-datepicker - Accessibility(Select missing label)

### ISSUE THREAD

**Issue by Spartan318** [ng-lightning/ng-lightning#388]
My ng-lightning version is 2.0.1, and I am using the latest version of Chrome. Based on the salesforce lightning design system notes for developers on datepickers: https://www.lightningdesignsystem.com/components/datepickers/
The ngl-datepicker should have a label element (with text 'Pick a Year') with a for attribute pointing to the id of the select element involved with picking years. This is the markup shown on the salesforce datepicker link provided: 
```
 <label class="slds-assistive-text" for="select-01">Pick a Year</label>
          <div class="slds-select_container">
            <select class="slds-select" id="select-01">
              <option>2014</option>
              <option>2015</option>
              <option>2016</option>
            </select>

```
This is how it is currently implemented on ng-lightning for ngl-datepicker:
```
<div class="slds-select_container">
<select class="slds-select" `ng-reflect-model="2018">`
```
There is no label and without this the ngl-datepicker is not accessible per the WCAG 2.1 guidelines.
Specifically, this violates success criterion 3.3.2 of WCAG 2.1, linked here:
https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html
The fix for this which is how salesforce says to implement the datepicker in their developer notes(linked at the top), is recommended as a sufficient technique for html in the link just above (Under sufficient techniques, number 2) and the link to that is: 
https://www.w3.org/WAI/WCAG21/Techniques/html/H44
This is a level A success criterion under WCAG 2.1 and I would like this simple fix implemented so I do not have to extend ng-lightning classes just to add this fix. 

Thanks,
Devin

**Comment by kavees1** [comment]
Ran into this issue as well is there a fix for it?
